kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ProduceActionIntegrationTest.java

```diff
@@ -82,13 +87,52 @@ public class ProduceActionIntegrationTest {
   private static final String DEFAULT_VALUE_SUBJECT = "topic-1-value";
 
   @RegisterExtension
-  public final DefaultKafkaRestTestEnvironment testEnv = new DefaultKafkaRestTestEnvironment();
+  public final DefaultKafkaRestTestEnvironment testEnv = new DefaultKafkaRestTestEnvironment(false);
 
   @BeforeEach
-  public void setUp() throws Exception {
+  public void setUp(TestInfo testInfo) throws Exception {
+    Properties restConfigs = new Properties();
+    if (testInfo.getDisplayName().contains("CallerIsRateLimited")) {
```

Comment:
This adds ability to set custom KafkaRestConfig for each test-method/use-case below.

Reply:
Could you add the comment to the code?